src/ui/components/pages/swap/swap.tsx

```diff
@@ -86,7 +88,7 @@ export function Swap({ setTab }: PageComponentProps) {
         assetId,
         assets[assetId],
       ]),
-    [assets]
+    [assets, swappableAssetIds.mainnet]
```

Comment:
`swappableAssetIds` is sufficient for dependencies.

Reply:
fixed